sonic_installer/bootloader/aboot.py

```diff
@@ -0,0 +1,106 @@
+
+import collections
+import os
+import re
+import subprocess
+
+import click
+
+from ..common import (
+   HOST_PATH,
+   IMAGE_DIR_PREFIX,
+   IMAGE_PREFIX,
+   run_command,
+)
+from .bootloader import Bootloader
```

Comment:
Is it possible to move common header lines into `__init__.py` ?

Reply:
There wouldn't be any advantage of doing so since the import would still be necessary for every python file. In the end you need to import what you need having import indirection doesn't really simplify anything and makes code browsing more cumbersome.